src/frontend/admin/src/components/presentational/menu-popover/MenuPopover.spec.tsx

```diff
@@ -1,31 +1,16 @@
 import { render, screen } from "@testing-library/react";
 import Button from "@mui/material/Button";
-import ListItemText from "@mui/material/ListItemText";
-import MenuItem from "@mui/material/MenuItem";
-import MenuList from "@mui/material/MenuList";
 import userEvent from "@testing-library/user-event";
-import {
-  MenuPopover,
-  useMenuPopover,
-} from "@/components/presentational/menu-popover/MenuPopover";
+import { MenuPopover } from "@/components/presentational/menu-popover/MenuPopover";
 
 describe("<MenuPopover />", () => {
   function TestMenu() {
-    const menu = useMenuPopover();
     return (
-      <>
-        <Button onClick={menu.open}>Open</Button>
-        <MenuPopover open={menu.anchor} onClose={menu.close} arrow="right-top">
-          <MenuList>
-            <MenuItem>
-              <ListItemText>One</ListItemText>
-            </MenuItem>
-            <MenuItem>
-              <ListItemText>Two</ListItemText>
-            </MenuItem>
-          </MenuList>
-        </MenuPopover>
-      </>
+      <MenuPopover
+        button={<Button>Open</Button>}
+        menuItems={[{ title: "One" }, { title: "Two" }]}
```

Comment:
Let's declare all existing props in order to detect regression about that later.
```
menuItems={[{ title: "One", onClick: noop, icon: <AbcIcon /> }, { title: "Two" }]}
```